.infrastructure/98_variables.tf

```diff
@@ -24,4 +24,16 @@ variable "tags" {
 ### required for security group ALB, ECS and RDS
 variable "cms_app_port" {
   default = 1337
+}
+
+### required for Strapi Site on AWS
+variable "dns_domain_name" {
+  description = "DNS domain for the B2B Portal product"
+  type        = string
+}
+
```

Comment:
```suggestion
```

Reply:
with next commit I comment the lines now to be deleted but necessary as soon as we have the domain and certificate

Reply:
changes present in the latest commit